charts/argocd-apps/values-integration.yaml

```diff
@@ -726,3 +726,39 @@ applications:
           secretKeyRef:
             name: signon-token-manuals-frontend-publishing-api
             key: bearer_token
+- name: email-alert-frontend
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/email-alert-frontend
+      tag: latest  # To be edited by automation (not yet implemented).
+    replicaCount: 1
+    extraEnv:
+      - name: SUBSCRIPTION_MANAGEMENT_ENABLED
```

Comment:
Might be worth adding a TODO to remind us that this one shouldn't be copied over to prod (it's for integration/staging only).